manager/allocator/cnmallocator/networkallocator.go

```diff
@@ -108,9 +125,8 @@ func New(pg plugingetter.PluginGetter, netConfig *NetworkConfig) (networkallocat
 		nodes:    make(map[string]map[string]struct{}),
 	}
 
-	// There are no driver configurations and notification
-	// functions as of now.
-	reg, err := drvregistry.New(nil, nil, nil, nil, pg)
+	// FIXME(thaJeztah): drvregistry.New was deprecated in https://github.com/moby/moby/commit/5595311209cc915e8b0ace0a1bbd8b52a7baecb0, but there's no other way to pass a PluginGetter to it.
```

Comment:
You're damn right there is no way to pass a PluginGetter when constructing the non-deprecated `drvregistry` types. That's because they don't need one. See also moby/moby@28edc8e2d692cb47f1ec6bb32a6e346777e1250d